x-pack/plugins/cases/server/client/configure/validators.ts

```diff
@@ -28,7 +28,7 @@ export const validateCustomFieldTypesInRequest = ({
     const originalField = originalCustomFields.find((item) => item.key === requestField.key);
 
     if (originalField && originalField.type !== requestField.type) {
-      invalidFields.push(requestField.key);
+      invalidFields.push(`'${requestField.label}'`);
```

Comment:
Usually, dynamic fields are surrounded by `"` in messages returned by APIs in Kibana.

Reply:
Fixed